geo_concerns/app/models/concerns/geo_concerns/image_behavior.rb

```diff
@@ -0,0 +1,31 @@
+module GeoConcerns
+  module ImageBehavior
+    extend ActiveSupport::Concern
+    include ::CurationConcerns::GenericWorkBehavior
+    include ::CurationConcerns::BasicMetadata
+    include ::GeoConcerns::BasicGeoMetadata
+
+    included do
+      aggregates :rasters, predicate: RDF::Vocab::ORE.aggregates,
+                           class_name: 'GeoConcerns::Vector',
+                           type_validator: type_validator
+      filters_association :members, as: :image_files, condition: :concerns_image_file?
```

Comment:
I think it's a namespace issue. If you look at the [raster model](https://github.com/jrgriffiniii/pcdm-geo-models/blob/73f400f4ebd0be0ff3c2c92b5bb0c0f0047e8c1d/geo_concerns/app/models/raster.rb), it isn't in the GeoConcerns namespace.